cmake/tests/have_pqcmdtuples.c

```diff
@@ -0,0 +1,9 @@
+#include <libpq-fe.h>
```

Comment:
`PQCMDTUPLES` is not used anymore, we can drop this file

Reply:
If so, we can drop it. Not sure about the history of this file and related checks.